WPMedia/engine-theme-sdk
# PEN-438 nativo ad integration

## Description
added nativo ad integration

## Jira Ticket
- [PEN-438](https://arcpublishing.atlassian.net/browse/PEN-438)

## Acceptance Criteria
- A new boolean Theme Setting called nativoIntegration should be created. 
  - If nativoIntegration = true, then Nativo functionality as described below should be added to the site
  - If nativoIntegration = false or if it’s not present at all, then no Nativo functionality should be added. 
- If nativoIntegration = true, then the Nativo script should be added to all pages on the site, per Nativo’s instructions: https://pub-resources.nativo.com/docs/javascript-integration-process
- We’ll add a new page-type value called nativo-clp. If page-type=nativo-clp on a given page, then the following should happen, per Nativo’s instructions https://pub-resources.nativo.com/docs/javascript-integration-process#31-add-javascript-and-meta-tags-to-clp
- Nativo script is included in the page
  -All metadata is removed from this page, except for the following which are added (and should only be added to nativo-clp pages: 
  - `<meta http-equiv="X-UA-Compatible" content="IE=edge" />`
  - `<meta name="robots" content="noindex, nofollow" />`
- See existing Page Types documentation: https://corecomponents.arcpublishing.com/alc/arc-products/themes/user-docs/page-template-guide/page-types/

## Test Steps
- when the new site's property `nativoIntegration` is added to a site, the Nativo script must be inserted on the page header for that site
- create a new page template and add the meta values `page-type: nativo-clp`
  - when this page is rendered, no social metadata must be added to the page and two meta tags must be inserted per Nativo request, check AC

## Effect Of Changes
### Before
is new functionality

### After
<img width="793" alt=" 2020-11-17-12 47 49" src="https://user-images.githubusercontent.com/9757/99414061-3064f580-28d5-11eb-86b5-fad6a9501a1e.png">

## Dependencies or Side Effects
- a new property on FP blocks.json site properties labeled `nativoIntegration: true`
- PR from `fusion-news-themes-blocks` https://github.com/WPMedia/fusion-news-theme-blocks/pull/561

## Review Checklist
please explain why (so that we can fix it whenever it gets refactored).
